libsignal-service-actix/src/websocket.rs

```diff
@@ -67,12 +67,13 @@ where
     let mut ka_interval = actix::clock::interval_at(
         actix::clock::Instant::now(),
         push_service::KEEPALIVE_TIMEOUT_SECONDS,
-    )
-    .fuse();
+    );
 
     loop {
+        let tick = ka_interval.tick().fuse();
+        futures::pin_mut!(tick);
         futures::select! {
-            _ = ka_interval.next() => {
+            _ = tick => {
```

Comment:
@gferon What do you think about this? https://github.com/tokio-rs/tokio/commit/7f17822ed9943b7bd0703258268feb4f70edac51Adds a stream implementation again, but this may be cleaner...

Reply:
well, this is what's supposed to land as `std::future::Stream` rather sooner than later, so it's not part of tokio 1.0, or did I misunderstand what you meant?